contracts/rust/src/bin/diff_test.rs

```diff
@@ -35,6 +34,7 @@ use jf_primitives::signatures::schnorr::KeyPair as SchnorrKeyPair;
 use jf_relation::{Arithmetization, Circuit, PlonkCircuit};
 use num_bigint::BigUint;
 use num_traits::Num;
+use std::str::FromStr;
```

Comment:
```suggestion
```
also remove the empty line above (L13)

Reply:
Looks like we need this import. Otherwise we get the following compilation error:

```
 just sol-test
cargo build --bin diff-test --release
   Compiling hotshot_contract v0.1.0 (/home/leloup/repositories/espressosys/espresso-sequencer/contracts/rust)
error[E0405]: cannot find trait `FromStr` in this scope
   --> contracts/rust/src/bin/diff_test.rs:574:6
    |
574 | impl FromStr for ParsedTranscript {
    |      ^^^^^^^ not found in this scope
    |
help: consider importing one of these items
    |
1   + use ark_std::str::FromStr;
    |
1   + use core::str::FromStr;
    |
1   + use std::str::FromStr;
    |

error[E0405]: cannot find trait `FromStr` in this scope
   --> contracts/rust/src/bin/diff_test.rs:629:6
    |
629 | impl FromStr for ParsedVerifyingKey {
    |      ^^^^^^^ not found in this scope
    |
help: consider importing one of these items
    |
1   + use ark_std::str::FromStr;
    |
1   + use core::str::FromStr;
    |
1   + use std::str::FromStr;
....
```

The empty line has been removed in 202638475ff911f53406be798c63a07f744a9ad5

Reply:
I see! I forgot that I have moved those to a different library file in my own PR i think. let's leave it then, I will deal with merge conflict in my PR later.